Content.Server/_Goobstation/Religion/WeakToHolySystem.cs

```diff
@@ -0,0 +1,122 @@
+// SPDX-FileCopyrightText: 2025 Aiden <28298836+Aidenkrz@users.noreply.github.com>
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Solstice <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 TheBorzoiMustConsume <197824988+TheBorzoiMustConsume@users.noreply.github.com>
+// SPDX-FileCopyrightText: 2025 gluesniffler <159397573+gluesniffler@users.noreply.github.com>
+// SPDX-FileCopyrightText: 2025 gluesniffler <linebarrelerenthusiast@gmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using System.Linq;
+using Content.Shared._Goobstation.Religion;
+using Content.Shared._Goobstation.Bible;
+using Content.Shared.Damage;
+using Content.Shared.Interaction;
+using Content.Shared.Inventory;
+using Robust.Shared.Physics.Events;
```

Comment:
Unused using directives
```suggestion
using Content.Shared._Goobstation.Bible;
using Content.Shared.Damage;
using Content.Shared.Interaction;
using Content.Shared.Inventory;
```